src/selectors/nodes.js

```diff
@@ -49,6 +53,18 @@ export const getNodeSelected = createSelector(
     )
 );
 
+/**
+ * Gets a map of nodeIds to nodes
+ */
+export const getNodesById = createSelector(
+  [state => state.graph.nodes],
```

Comment:
Conversely, I'd make this a named function (`getGraphNodes`?) and put it up top

Reply:
Okay I think that works if we make the distinction, since that seems to be the case elsewhere, so made the change.